[Code & Docs] Add Support for Steam Deck (#19)
imLinguin/comet: README.md

comet --from-lutris --username <USERNAME>
```

> Or use the shortcut script provided for non-Steam shortcuts. See [the installation instructions](steamdeck/USAGE.md). This is useful for Steam Deck users.

Reviewer: This reads like it is linking to generic instructions that may be useful to Steam Deck users. Maybe it could be reworded something like:

```suggestion
Or use the shortcut script provided for non-Steam shortcuts. See the [Steam Deck Usage Guide](steamdeck/USAGE.md).
```
